invenio_communities/views/communities.py

```diff
@@ -233,3 +233,33 @@ def invitations(community=None, pid_value=None):
         },
         permissions=permissions,
     )
+
+
+@pass_community
+@pass_invitation
+def invitation_details(community=None, invitation=None, pid_value=None, invitation_id=None):
+    query_config = dict(
+        size=current_app.config['COMMUNITIES_INVITATION_DETAILS_TIMELINE_PAGE_SIZE']
+    )
+
+    permissions = community.has_permissions_to(
+        ['update', 'read', 'search_requests', 'search_invites']
+    )
+    if not permissions['can_search_invites']:
+        raise PermissionDeniedError()
+
+    return render_template(
+        "invenio_communities/details/members/invitation_details.html",
+        community=community.to_dict(),
+        types={
+            "organization": "Organization",
+            "event": "Event",
+            "topic": "Topic",
+            "project": "Project"
+        },
+        permissions=permissions,
```

Comment:
it is hard to see because we have 2 layers of template inheritance but if you look inside these templates https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/templates/semantic-ui/invenio_communities/details/members/base.html and https://github.com/inveniosoftware/invenio-communities/blob/master/invenio_communities/templates/semantic-ui/invenio_communities/details/header.html you can see where they are used